pure2.md

```diff
@@ -0,0 +1,535 @@
+                "keyids":
+                [
+                    "d31bdd6fe24de1d0e865ef924ea2d23724721d4b9f338d2a93379d14aa1e953f"
+                ],
+                "threshold": 1
+            },
+            "root":
+            {
+                "keyids":
+                [
+                    "e32270db69268abb6b709266679a065a23cfd4351d9cda63b5537195dfbab14d"
+                ],
+                "threshold": 1
+            }
+        },
+        "version": 5,
+        "expires": "2022-04-30T12:04:47Z"
+    }
+}
+```
+
+## Deployment Considerations
+
+### Conditions for offline-update acceptance
+
+PURE-2 does not specify when offline updates should be accepted by devices; this is left to the discretion of implementors. For example, an implementation might allow their diagnostic tool to start the Uptane client in a special mode that permits offline updates, only checking the well-known location for offline-update metadata under that condition. If the only desired use for offline updates is during factory provisioning, an implementation might choose to disable offline updates as soon as the device registers its ECU keys with the inventory database for the first time.
+
+In general, we recommend being conservative and careful about the scenarios in which offline updates are accepted, because of the risk that an attacker could find a way to exploit the well-known location and install an undesirable (but not-yet-expired) offline update.
+
+### Preparing an ECU for PURE-2
+
+The prerequisites for PURE-2 updates are similar to the prerequisites for normal Uptane updates, except that the ECU does not need to have an ECU key yet. This follows from the fact that no ECU manifests or vehicle version reports are created during an offline update, and is useful in factory programming use cases, where ECUs may not yet have their private key material provisioned in before receiving a software update.
+
+### Partial-verification ECUs and PURE-2
+
+It is possible for a partial-verification ECU to perform PURE-2 offline updates. Like partial verification for normal Uptane updates, a partial-verification ECU MAY verify only the offline update targets metadata file it receives from the primary.
+
+### Metadata expiry times
+
+The Offline Updates role signs two different types of metadata: offline-update targets metadata files, which contain the targets to be installed, and the the offline-update snapshot metadata, which lists the currently-valid offline-update targets metadata files. For an offline update to be installed, both the offline update snapshot metadata file and the targets metadata file being installed MUST be within their validity period.
+
+Therefore, choosing the expiry times for these files is an important security consideration, and an important usability consideration. Making the right choice will depend heavily on the implementation and what purpose offline updates are serving. For example, if offline updates are designed for 
```

Comment:
Eh?

Reply:
Whoops, looks like the rest of this paragraph got lost in the ether somewhere. Will finish it up shortly.

Reply:
fixed...

> Will finish it up shortly.

....on Nov. 23. That still counts as "shortly" on a standards-approval timescale, right? :-/